applications/2025/pending/subhopam-das-personal.yml

```
@@ -0,0 +1,113 @@
+    monitoring_tools:  # Datadog, Sentry, etc.
+      - Grafana
+      - Prometheus
+    ci_cd_experience:  # GitHub Actions, etc.
+      - github actions
+      - docker
+      - kubernetes
+      - aws
+    biggest_prod_userbase: 300K users across the globe # Largest audience you've shipped to?
+
+shipping_velocity:
+  fastest_feature_shipped: 
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)